src/popup/utils/constants.js

```diff
@@ -369,3 +369,7 @@ export const DEX_TRANSACTION_TAGS = {
   deposit: DEX_SWAP,
   withdraw: DEX_SWAP,
 };
+
+export const SYMBOLS = {
+  AE: 'AE',
+};
```

Comment:
I think, if we now decided to use a symbol for AE (will there be more of them or AE symbol should be a standalone constant?) it should be used in whole app